Sources/HaCWebsiteLib/Utils/DatabaseUtils.swift

```diff
@@ -28,34 +28,36 @@ extension URL {
   }
 }
 
-func testDatabase() {
-  do {
-    guard let databaseURLString = DotEnv.get("DATABASE_URL"),
-      let databaseURL = URL(string: databaseURLString),
-      let databaseURLComponents = databaseURL.databaseURLComponents else {
-      fatalError("Couldn't find valid DATABASE_URL environment variable")
-    }
+func getDatabaseDriver() throws -> PostgreSQLDriver.Driver {
```

Comment:
It appears that this is only used in this file. To this effect, could we make it private?